views/messages.erb

```diff
@@ -0,0 +1,30 @@
+ <div style="text-align:center;">   
+    <h1 style="color:#5cd65c;font-family:courier;font-size:350%;"> Chitter </h1>
+
+    <form action="/messages" method="post">
+        <input style="font-family:courier;padding:13px 80px;text-align:center;font-size:16" type="text" name="new_message" placeholder="New Peep" maxlength="60" />
```

Comment:
instead of using inline styling, you could add a <style> tag at the top of your documents and add all of your styling in there to keep your code clean/readable, whilst maintaining the styling elements you have added - without using CSS :) 
(if you are using multiple h1 tags and want to style them each differently for example, you could achieve this by adding an "id" to each h1 tag and then reference that id within the <style></style> tag - not sure if this makes sense).